src/main/config/config.h

```diff
@@ -51,6 +51,7 @@ typedef struct systemConfig_s {
     uint8_t hseMhz;                 // Only used for F4 and G4 targets
     uint8_t configurationState;     // The state of the configuration (defaults / configured)
     uint8_t enableStickArming; // boolean that determines whether stick arming can be used
+    uint8_t statsSaveMoveLimit; // gyro rate limit for saving stats upon disarm
```

Comment:
Same for this one. You can more it to [statsConfig_t](https://github.com/betaflight/betaflight/blob/master/src/main/pg/stats.h)